apps/communications/contacts/js/views/list.js

```diff
@@ -833,15 +855,24 @@ contacts.List = (function() {
 
     // Replacing old message 'startup-path-done'
     utils.PerformanceHelper.loadEnd();
-    fb.init(function contacts_init() {
-      if (fb.isEnabled) {
-        Contacts.loadFacebook(NOP_FUNCTION);
-      }
-      lazyLoadImages();
-      loaded = true;
+    LazyLoader.load([
+     '/contacts/js/fb/fb_init.js',
+     '/contacts/js/fb_loader.js',
+     '/shared/js/contacts/utilities/image_loader.js'
+    ], () => {
+      fb.init(function contacts_init() {
+        if (fb.isEnabled) {
+          Contacts.loadFacebook(NOP_FUNCTION);
+        }
+        lazyLoadImages();
+        loaded = true;
+      });
     });
-
     loadICE();
```

Comment:
I'm just thinking, should we make `loadICE` return a promise when done (if we have ice or not), and cache there? With that we will be sure that we are caching once we have all the list rendered.